gym_roboy/example_control.py

```diff
@@ -1,34 +1,30 @@
-import gym
-import time
+import os
+
 from stable_baselines.common.policies import MlpPolicy
 
 from stable_baselines.common.vec_env import DummyVecEnv
 from stable_baselines import PPO1
 
-from .envs import MsjEnv, MsjROSBridgeProxy
+from .envs import MsjEnv
 
 
-def our_env_constructor() -> gym.Env:
-    return MsjEnv(MsjROSBridgeProxy())
+MOUNT_DIR = "/root/develDeepAndReinforced/tensorboard_out/"
+MODEL_FILE = os.path.join(MOUNT_DIR, "model")
```

Comment:
Let's think of a more meaningful name which involves ppo1 as well. Might be useful if we want to compare the performance of different algorithms.

Reply:
sure, any suggestion? What about 
```python
MODEL_FILE = os.path.join(MOUNT_DIR, "2019_02_03_PPO1")
```

Reply:
it could also involve something like `str(datetime.now())`

Reply:
ok